CMakeLists.txt

```diff
@@ -59,5 +59,8 @@ if (${PROJECT_NAME}_TESTS)
     add_subdirectory(tests)
 endif()
 if (${PROJECT_NAME}_EXAMPLES)
+    find_package(QT NAMES Qt6 Qt5 COMPONENTS Widgets REQUIRED)
```

Comment:
I don't think it makes sense to find QT again.

These two lines should be simply `find_package(Qt${QT_VERSION_MAJOR}Widgets)`